ios/FluentUI/SegmentedControl/SegmentedControl.swift

```diff
@@ -417,8 +417,11 @@ open class SegmentedControl: UIView, TokenizedControlInternal {
     private let selectionChangeAnimationDuration: TimeInterval = 0.2
 
     private func updateButtons() {
+        let contentColor = isEnabled ? UIColor(dynamicColor: tokenSet[.restLabelColor].dynamicColor) : UIColor(dynamicColor: tokenSet[.disabledLabelColor].dynamicColor)
         for (index, button) in buttons.enumerated() {
             button.updateTokenizedValues()
+            button.setTitleColor(contentColor, for: .normal)
+            button.tintColor = contentColor
```

Comment:
Don't we need to use `UIButton.configuration` for changing our colors now?

Reply:
We still haven't dropped 14, so we need both at the moment.